challenge.rb

```diff
@@ -6,7 +6,8 @@
 # and should return either `true` or `false` based on whether `age` is less than `minimum_age`.
 
 def is_of_age? age, minimum_age
-  # Replace this comment with your code.
+ check = (minimun_age.to_i) - age.to_i)
+ check >= 0
```

Comment:
Should be `true` if age is _greater_ than _minimum age_, but the question is worded a little ambiguously, so I won't take points off.